REQUIRE

```diff
@@ -1,3 +1,3 @@
 julia 0.7
 BinaryProvider 0.5
-BinDeps
+BinDeps
```

Comment:
Why this change?  Please, always keep the last line of a file ending with a newline, otherwise several tools, including git, would be unhappy.

Reply:
Okay adding the newline . Actually , i had previously put a dependency there of `StatsFuns` but later removed it . Since i needed only one single function and hence decided to copy it here instead of putting a dependency on that package.